clientd3d/drawbsp.c

```diff
@@ -23,7 +23,7 @@
 
 #include "client.h"
 
-#define MAX_OBJS_PER_LEAF 100
+#define MAX_OBJS_PER_LEAF 1000
```

Comment:
This is absurdly high; there is no reason to draw 1000 objects in one BSP leaf.  It could cause performance problems, though.